src/plugins/ui_core_views/cell_evaluation/evaluator.ts

```diff
@@ -241,6 +242,7 @@ export class Evaluator {
           this.evaluatedCells.set(position, evaluatedCell);
         }
       }
+      onCycleEndEvaluationRegistry.getAll().forEach((callback) => callback(this.getters));
```

Comment:
I'd rename "cycle" to "iteration". It's more precise IMO. "cycle" could be understood as 1) the full eval cycle 2 ) a cyclic dependency